routers/web/repo/pull.go

```diff
@@ -1449,6 +1449,17 @@ func CompareAndPullRequestPost(ctx *context.Context) {
 		return
 	}
 
+	if projectID > 0 {
+		if !ctx.Repo.CanWrite(unit.TypeProjects) {
+			ctx.Error(http.StatusBadRequest, "user hasn't the permission to write to projects")
+			return
+		}
+		if err := issues_model.ChangeProjectAssign(ctx, pullIssue, ctx.Doer, projectID); err != nil {
```

Comment:
Probably
```suggestion
		if err := issues_model.ChangeProjectAssign(pullIssue, ctx.Doer, projectID); err != nil {
```
if I had to guess

Reply:
Seen so.... can you apply the code suggestion? As it is a Giteabot branch, I cannot update it